### PR TITLE
build: update to Angular 11.1.0

### PR DESCRIPTION
**Pull request by crisbeto**
Bumps all of the Angular dependencies to 11.1.0 and TS 4.1.2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,6 @@
     "shelljs": "^0.8.4",
     "ts-node": "^8.10.2",
     "tslint": "^6.1.3",
-    "typescript": "4.0.2"
+    "typescript": "4.1.2"
```
Comment (Splaktar) [inline package.json:86]: `ng update` doesn't bump this, but I saw that it's supported. I guess we'll try it out and see if there are any issues.

File: package.json [hunk 2]
```diff
@@ -83,6 +83,6 @@
     "shelljs": "^0.8.4",
```
Comment (Splaktar) [inline package.json:83]: `ng update` also bumps up to `"karma": "~5.2.3",`

**Review by Splaktar — APPROVED** [review]
LGTM with one comment about karma.

File: package.json (hunk 1 above)
Comment (crisbeto) [reply package.json:86]: I bumped it since it's the version we're using on the main repo.

**Pull request merged**